sql/atxs/atxs_test.go

```diff
@@ -50,6 +50,32 @@ func TestGet(t *testing.T) {
 	require.ErrorIs(t, err, sql.ErrNotFound)
 }
 
+func TestAll(t *testing.T) {
+	db := sql.InMemory()
+
+	atxList := make([]*types.VerifiedActivationTx, 0)
+	for i := 0; i < 3; i++ {
+		sig, err := signing.NewEdSigner()
+		require.NoError(t, err)
+		atx, err := newAtx(sig, withPublishEpoch(types.EpochID(i)))
+		require.NoError(t, err)
+		atxList = append(atxList, atx)
+	}
+
+	expected := map[types.ATXID]struct{}{}
+	for _, atx := range atxList {
+		require.NoError(t, atxs.Add(db, atx))
+		expected[atx.ID()] = struct{}{}
+	}
+
+	all, err := atxs.All(db)
+	require.NoError(t, err)
+	for _, got := range all {
+		delete(expected, got)
+	}
+	require.Empty(t, expected)
```

Comment:
Why not simply use `require.ElementsMatch()`? For example:
```suggestion
	var expected []types.ATXID
	for _, atx := range atxList {
		require.NoError(t, atxs.Add(db, atx))
		expected = append(expected, atx.ID())
	}

	all, err := atxs.All(db)
	require.NoError(t, err)
	require.ElementsMatch(t, expected, all)
```